lua/default-config.lua

```diff
@@ -82,7 +75,7 @@ O = {
             autoformat = false,
             isort = false,
             diagnostics = {
-                virtual_text = {spacing = 0, prefix = ""},
+                virtual_text = {spacing = 0, prefix = "⎨"},
```

Comment:
ooops, this was not intended to be in this commit... I can remove it if you prefer the previous default